### PR TITLE
nmp79/completed challenges photo feed 

### PR DESCRIPTION
**Pull request by nit0107**
### Summary 

This pull request includes the implementation of the photo feed for completed journeys/challenges on the profile page. 

- [x] implemented scrollable photo feed
- [x] implemented swipeable carousel that allows for multiple images
- [x] implemented stateful widget to use for multiple challenges
- [x] navigation from main profile page
- [x] created new screen to view all the challenges

Remaining TODOs:

- [ ] connect with backend

### Test Plan 
Widgets were called with prefilled data, so that once the dto is connected, it can easily send values to the corresponding widget

image

image

### CODE REVIEW

File: game/lib/preview/preview.dart [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import 'package:flutter/material.dart';
-import 'package:flutter/widgets.dart';
 import 'package:game/gameplay/gameplay_page.dart';
 
 enum previewType { challenge, journey }
```
Comment (neketka) [inline game/lib/preview/preview.dart:4]: Nit: PreviewType instead of previewType

Comment (neketka) [reply game/lib/preview/preview.dart:4]: Also CHALLENGE and JOURNEY

File: package.json [hunk 2]
```diff
@@ -13,5 +13,8 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11"
+  },
```
Comment (neketka) [inline package.json:16]: We already have this in the server folder and it doesn't look like you've added any scripts. Is this necessary?

**Review by neketka — CHANGES_REQUESTED** [review]
Really good progress, check out some of my comments

**Comment by jasmineee-li** [comment]
Tried out on my end -- this looks great & matches well w/the handed-off designs! Nikita catches some good things above

**Comment by jasmineee-li** [comment]
checked CompletedChallengesPage() on my end after the typo fix; works for me! nice job

**Review by jasmineee-li — APPROVED** [review]

**Pull request merged**